.github/workflows/windows.yml

```diff
@@ -0,0 +1,30 @@
+name: Windows build
+
+on: [push, pull_request]
+
+jobs:
+  build:
```

Comment:
Do we need to update `mergify` config (we might want to rename `build` to say `win_build` to avoid collision)?

Reply:
Fixed in https://github.com/pystatgen/sgkit/pull/269/commits/5314a835299b1612a0928e0c33eeb28055afdcfb